src/base.css

```diff
@@ -1344,6 +1346,19 @@ Possible extra rowspan handling
 		font-weight: bold;
 	}
 
+/** Character examples (from the I18N template) *******************************/
```

Comment:
I'd suggest the following changes to the code.
````

.codepoint bdi {
   font-size: 120%;
   }
.codepoint .uname {
   font-size: 80%;
   margin-inline: 2px;
   letter-spacing: 0.03em;
   text-transform: uppercase;
   }
````

Additional, document-specific CSS styling may be necessary to style the contents of the bdi, usually based on the lang attribute attached to that element. This would apply font-family, and possibly (if the font has unusual metrics) font-size settings.

Reply:
Done!

Reply:
@r12a are you able to merge this?